### PR TITLE
DS-154: Refactor AvalancheGo to build/maintain/validate sections

### PR DESCRIPTION
**Pull request by yulin-dong**

### CODE REVIEW

File: docs/build/tutorials/avalanchego-node/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+
+## Maintain
+|  |  |
+| :--- | :--- |
+| [**Set up Node Monitoring**](./maintain/setting-up-node-monitoring.md) | Set up infrastructure to monitor an instance of AvalancheGo |
+| [**Node Backup and Restore**](./maintain/node-backup-and-restore.md) | Back up important files to be able to restore your node |
+| [**Upgrade Your AvalancheGo Node**](./maintain/upgrade-your-avalanchego-node.mdx) | Upgrade your Avalanche node |
+| [**Run C-Chain Offline Pruning**](./maintain/run-offline-pruning.md) | C-Chain offline pruning |
+| [**Hetzner Volume Migration**](./maintain/hetzner-volume-migration.md) | Hetzner Volume Migration |
```
Comment (aaronbuchwald) [inline docs/build/tutorials/avalanchego-node/README.md:26]: Can we remove this? I don't think it makes sense to have highly specific documentation for this case anymore.

File: docs/build/avalanchego/avalanchego-apis/keystore.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ Every node has a built-in keystore. Clients create users on the keystore, which
 
 _**You should only create a keystore user on a node that you operate, as the node operator has access to your plaintext password.**_
 
-For validation and delegation on main net, you should issue transactions through [the wallet](../../tutorials/nodes-and-staking/staking-avax-by-validating-or-delegating-with-the-avalanche-wallet.md). That way control keys for your funds won't be stored on the node, which significantly lowers the risk should a computer running a node be compromised.
+For validation and delegation on main net, you should issue transactions through [the wallet](../../tutorials/avalanchego-node/validate/staking-avax-by-validating-or-delegating-with-the-avalanche-wallet.md). That way control keys for your funds won't be stored on the node, which significantly lowers the risk should a computer running a node be compromised.
```
Comment (aaronbuchwald) [inline docs/build/avalanchego/avalanchego-apis/keystore.md:10]: nit: mainnet should be one word.

**Comment by aaronbuchwald** [comment]
Can we put the postman collection within the AvalancheGo APIs section?

Also, can we change the phrase AvalancheGo Server Software to either just AvalancheGo or AvalancheGo Node?

File: docs/build/tutorials/avalanchego-node/README.md (hunk 1 above)
Comment (yulin-dong) [reply docs/build/tutorials/avalanchego-node/README.md:26]: @jpop32 what is your opinion on this?

Comment (yulin-dong) [reply docs/build/tutorials/avalanchego-node/README.md:26]: Let me see where I can put it

**Pull request merged**